### PR TITLE
CMR-8734: As a CMR developer, I want to migrate OptionDefinition in Legacy Services to CMR core in SIT

### PR DESCRIPTION
**Pull request by TylerHeald1**

### CODE REVIEW

File: legacy-migration/migrate.py [hunk 1]
```diff
@@ -99,6 +119,21 @@ def migrate_data_quality_summary():
...
+    print("Starting OptionDefinition data migration...")
+
+    with oracledb.connect(user=user, password=pwd, dsn="localhost:1521/orcl") as connection:
+        with connection.cursor() as cursor:
+            sql = """select GUID, NAME, DESCRIPTION, FORM, SCOPE, SORT_KEY, DEPRECATED, PROVIDER_GUID from I_10_0_TESTBED_BUSINESS.EJB_OPTION_DEF"""
+            for r in cursor.execute(sql):
+                migrate_od_row(r[0], r[1], r[2], r[3], r[4], r[5], r[6], r[7])
+
+    end = time.time()
+    print(f"Total OptionDefinition data migration time spent: {end - start} seconds, Succeeded: {success_count}, Failed: {failure_count}")
```
Comment (eereiter) [inline legacy-migration/migrate.py:136]: print(f"Total - is this correct?

Comment (TylerHeald1) [reply legacy-migration/migrate.py:136]: The f is for a formatted string to include the {end - start} time as a calculated value, and the {success_count} and {failure_count}

**Review by eereiter — APPROVED** [review]

File: legacy-migration/migrate.py [hunk 2]
```diff
@@ -13,6 +13,10 @@
 pwd = os.environ.get('MIGRATE_PWD')
 access_token = os.environ.get('ACCESS_TOKEN')
 
+header = {"Content-Type": "application/json",
+              "Authorization": access_token,
```
Comment (jaybarra) [inline legacy-migration/migrate.py:17]: Spacing should be fixed here

File: legacy-migration/migrate.py [hunk 3]
```diff
@@ -125,3 +160,4 @@ def migrate_data_quality_summary_assignment():
 provider_guid_to_name_map()
 migrate_data_quality_summary()
 migrate_data_quality_summary_assignment()
+migrate_option_definition()
```
Comment (jaybarra) [inline legacy-migration/migrate.py:163]: Should update editor settings to ensure newlines at the end of files.

**Review by jaybarra — APPROVED** [review]

**Pull request merged**